5/debian-10/rootfs/opt/bitnami/scripts/libphpmyadmin.sh

```diff
@@ -91,6 +91,19 @@ phpmyadmin_initialize() {
         ! is_empty_value "$DATABASE_SSL_VERIFY" && phpmyadmin_conf_set "\$cfg['Servers'][\$i]['ssl_verify']" "$(php_convert_to_boolean "$DATABASE_SSL_VERIFY")" yes
     fi
 
+    if ! is_empty_value "$CONFIGURATION_STORAGE_DATABASE_CONTROLUSER" &&
+       ! is_empty_value "$CONFIGURATION_STORAGE_DATABASE_CONTROLPASS" &&
+       ! is_empty_value "$CONFIGURATION_STORAGE_DATABASE_CONTROLHOST" &&
+       ! is_empty_value "$CONFIGURATION_STORAGE_DATABASE_CONTROLPORT" &&
+       ! is_empty_value "$CONFIGURATION_STORAGE_DATABASE_PMADB"; then
```

Comment:
I think that ignoring everything if a single variable is missing may lead to confusion.

I would suggest adding a variable, e.g `CONFIGURATION_STORAGE_ENABLE` that made it easier to control:

```suggestion
    if is_boolean_yes "$CONFIGURATION_STORAGE_ENABLE"; then
```

Then, in the `phpmyadmin_validate`, add a section like this:
```bash
    if is_boolean_yes "$CONFIGURATION_STORAGE_ENABLE"; then
        for empty_env_var in "CONFIGURATION_STORAGE_DATABASE_CONTROLHOST" "CONFIGURATION_STORAGE_DATABASE_CONTROLPORT" "CONFIGURATION_STORAGE_DATABASE_CONTROLUSER" "CONFIGURATION_STORAGE_DATABASE_CONTROLPASS"
"CONFIGURATION_STORAGE_DATABASE_PMADB"; do
            is_empty_value "${!empty_env_var}" && print_validation_error "The ${empty_env_var} environment variable is empty or not set."
        done
  fi
```

Reply:
Done that. Please review.